src/orders/components/OrderRefundAmount/OrderRefundAmount.tsx

```diff
@@ -1,23 +1,123 @@
-import { makeStyles } from "@material-ui/core";
+import { makeStyles, RadioGroup } from "@material-ui/core";
 import Button from "@material-ui/core/Button";
 import Card from "@material-ui/core/Card";
 import CardContent from "@material-ui/core/CardContent";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Radio from "@material-ui/core/Radio";
 import Typography from "@material-ui/core/Typography";
+import CardSpacer from "@saleor/components/CardSpacer";
 import CardTitle from "@saleor/components/CardTitle";
-import Money, { addMoney, subtractMoney } from "@saleor/components/Money";
+import ControlledCheckbox from "@saleor/components/ControlledCheckbox";
+import Hr from "@saleor/components/Hr";
+import { addMoney, IMoney, subtractMoney } from "@saleor/components/Money";
 import PriceField from "@saleor/components/PriceField";
-import Skeleton from "@saleor/components/Skeleton";
 import { OrderErrorFragment } from "@saleor/fragments/types/OrderErrorFragment";
 import { OrderRefundData_order } from "@saleor/orders/types/OrderRefundData";
 import { getFormErrors } from "@saleor/utils/errors";
 import getOrderErrorMessage from "@saleor/utils/errors/order";
 import React from "react";
 import { FormattedMessage, useIntl } from "react-intl";
 
-import { OrderRefundData } from "../OrderRefundPage/form";
+import OrderRefundAmountValues, {
+  OrderRefundAmountValuesProps
+} from "../OrderRefundAmountValues";
+import {
+  OrderRefundAmountCalculationMode,
+  OrderRefundFormData,
+  OrderRefundType
+} from "../OrderRefundPage/form";
+
+const getMiscellaneousAmountValues = (
+  order: OrderRefundData_order
+): OrderRefundAmountValuesProps => {
+  const authorizedAmount = order?.total?.gross;
+  const previouslyRefunded =
+    order?.totalCaptured &&
+    authorizedAmount &&
+    subtractMoney(order?.totalCaptured, authorizedAmount);
+  const maxRefund =
+    authorizedAmount &&
+    previouslyRefunded &&
+    addMoney(authorizedAmount, previouslyRefunded);
+
+  return {
+    authorizedAmount,
+    maxRefund,
+    previouslyRefunded
+  };
+};
+
+const getProductsAmountValues = (
+  order: OrderRefundData_order,
+  data: OrderRefundFormData
+): OrderRefundAmountValuesProps => {
+  const authorizedAmount = order?.total?.gross;
+  const shipmentCost =
+    authorizedAmount?.currency &&
+    (order?.shippingPrice?.gross || {
+      amount: 0,
+      currency: authorizedAmount?.currency
+    });
+  const previouslyRefunded =
+    order?.totalCaptured &&
+    authorizedAmount &&
+    subtractMoney(order?.totalCaptured, authorizedAmount);
+  const maxRefund = order?.totalCaptured;
+  const orderLinesSum = order?.lines?.reduce((sum, line) => {
```

Comment:
```suggestion
  const refundedLinesSum = order?.lines?.reduce((sum, line) => {
```

This name made me confused when reading this code.

Reply:
fixed